packages/bolt-storage-file/README.md

```diff
@@ -0,0 +1,25 @@
+# Bolt File Storage
```

Comment:
don't love the name "storage-file" for this package. no immediate suggestions - maybe something combination of `dev`, `store`, `persist`, `file`, `state`. like `bolt-dev-persist` or `bolt-state-store`. `bolt-file-store` sounds like you're storing files almost, though, which doesn't make a ton of sense.

Reply:
also, probably worth providing a simple helper to clean the persistent storage? maybe as a separate PR.

Reply:
> don't love the name "storage-file"

Same. My intent was to play prefix games for sorting and the like.

For example, in production we will likely need to write a "bolt-storage-postgres". I'm both surprised and not at all that I couldn't find any other existing art here. 

I could work `convo` store in there too, but like. That seems weird, I'm also trying to use it for interactions which isn't a convo technically?

Reply:
> probably worth providing a simple helper to clean the persistent storage? maybe as a separate PR.

Seems interesting, I'm not sure I see what dx your looking to help.

My current process is just `rm -rf .db`

Reply:
kind of a small thing, but that currently requires people both knowing how this is implemented and where the file is/its name.